tests/integration/auth.spec.coffee

```diff
@@ -136,6 +137,7 @@ describe 'SDK authentication', ->
 
 		describe 'resin.auth.getUserId()', ->
 
-			it 'should eventually be the user id', ->
-				promise = resin.auth.getUserId()
-				m.chai.expect(promise).to.eventually.equal(credentials.userId)
+			it 'should eventually be a user id', ->
+				resin.auth.getUserId()
+				.then (userId) ->
+					m.chai.expect(userId).to.be.a('number')
```

Comment:
So, it's a funny realization that this is the only place that `credentials.userId` was used.
I'm fine as it is, but since this is the so famous `userId`, would it make any sense to also cross-check it with a server returned value eg requesting the user with that exact username from the api?
Or at least, how about verifying that it's a positive integer?

Reply:
I think doing a separate pine request to confirm feels a bit like overkill imo. I've added a bonus check to make sure it's a positive integer though.

Reply:
Woohoo!